src/osd/ClassHandler.h

```diff
@@ -74,18 +72,19 @@ class ClassHandler
 
     ClassMethod *_get_method(const char *mname);
 
-    ClassData() : status(CLASS_UNKNOWN), 
-		  handler(NULL),
-		  handle(NULL) {}
-    ~ClassData() { }
+    ClassData() : status(CLASS_UNKNOWN), handler(nullptr), handle(nullptr) {}
```

Comment:
might want to take this opportunity to initialize these member variable in-class.

Reply:
Fixed.